geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -176,8 +176,20 @@
   private static final int prMetaDataSentCountId;
 
   private static final int localMaxMemoryId;
+  static final int clearLocalDurationId;
```

Comment:
I think clearLocalDuration and clearTotalDuration should no longer exist

Reply:
agreed.